### PR TITLE
Event subscription logic

### PR DESCRIPTION
**Pull request by firefinchdev**

### CODE REVIEW

File: src/routers/api/clients.js [hunk 1]
```diff
@@ -48,10 +61,65 @@ router.post('/edit/:id', cel.ensureLoggedIn('/login'),
...
                 options.webhookURL = req.body.webhookurl
-              }
+            }
             await updateClient(options, clientId)
+
+            await deleteEventSubscription (req.params.id)
+
+            let event_subscription = []
+            // --------------------- User ------------------------------ //
+            if (req.body.cUser) {
```
Comment (championswimmer) [inline src/routers/api/clients.js]: don't depend on capitalisation on urlencoded form fields, use `event_c_user`, `event_u_user` like this for the form fields

**Comment by firefinchdev** [comment]
@championswimmer 
Changes done.
Pls review.

**Comment by championswimmer** [comment]
fix the conflicts please. everything looks good otherwise

**Comment by firefinchdev** [comment]
@championswimmer 
Done. Pls review.

**Pull request merged**